### ISSUE TITLE
ProcessException: The system cannot find the file specified.

### ISSUE THREAD

**Issue by abughalib** [leancodepl/patrol#545]
Error
```powershell
❯ patrol doctor
artifact path: C:\Users\Abugh\.cache\patrol (default)
ProcessException: The system cannot find the file specified.
  Command: which adb
#0      _ProcessImpl._runAndWait (dart:io-patch/process_patch.dart:487:7)
#1      _runNonInteractiveProcessSync (dart:io-patch/process_patch.dart:632:18)
#2      Process.runSync (dart:io-patch/process_patch.dart:68:12)
#3      DoctorCommand._checkIfInstalled (package:patrol_cli/src/features/doctor/doctor_command.dart:69:31)
#4      DoctorCommand._printAndroidSpecifics (package:patrol_cli/src/features/doctor/doctor_command.dart:51:5)
#5      DoctorCommand.run (package:patrol_cli/src/features/doctor/doctor_command.dart:32:5)
#6      CommandRunner.runCommand (package:args/command_runner.dart:209:27)
#7      PatrolCommandRunner.runCommand (package:patrol_cli/src/command_runner.dart:233:30)
#8      PatrolCommandRunner.run (package:patrol_cli/src/command_runner.dart:184:24)
#9      patrolCommandRunner (package:patrol_cli/src/command_runner.dart:45:27)
<asynchronous suspension>
#10     main (file:///C:/Users/Abugh/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/patrol_cli-0.7.4/bin/main.dart:6:20)
<asynchronous suspension>
```
Configurations.
* adb is already added to the Path.
* `ANDROID_HOME` is defined in an environment variable.

**Comment by bartekpacia** [comment]
This is caused by the `which` being unavailable on Windows. I introduced this bug in #541.

**Comment by abughalib** [comment]
I guess I would use the CLI on WSL or Linux.

**Comment by bartekpacia** [comment]
Sure, you could try this workaround.

If you want you can also contribute a fix. [Here's the code that causes this problem](https://github.com/leancodepl/patrol/blob/master/packages/patrol_cli/lib/src/features/doctor/doctor_command.dart#L68-L76).

**Comment by bartekpacia** [comment]
Fixed in #546
